compliance_checker/tests/test_cf_integration.py

```diff
@@ -323,9 +323,9 @@ def test_pr_inundation(self):
             u'§2.6.2 depth:comment should be a non-empty string', 
             u'§2.6.2 institution global attribute should be a non-empty string', 
             u'§2.6.2 comment global attribute should be a non-empty string', 
-            u"'units' attribute of 'LayerInterf' must be a string compatible with UDUNITS", 
-            u"'units' attribute of 'time_bounds' must be a string compatible with UDUNITS", 
-            u"'units' attribute of 'Layer' must be a string compatible with UDUNITS", 
```

Comment:
let's keep this text, "compatible with UDUNITS". It tells the user a little extra so hopefully they can fix the units in one shot.